src/type.c

```diff
@@ -856,7 +856,7 @@ EcsTypeComponent type_from_expr(
     const char *expr)
 {
     ecs_vector_t *vec = ecs_vector_new(&handle_arr_params, 1);
-    ecs_parse_component_expr(world, expr, parse_type_action, &vec);
+    ecs_parse_component_expr(world, expr, parse_type_action, NULL, &vec);
```

Comment:
I don't like this (NULL in the system_id). What do you think? Can you think in a better way to do it??

Reply:
I think for now it's ok. Perhaps in a later revision `system_id` is not the right name, and it should be something more generic like `name`.